### ISSUE TITLE
DOC: cvxpy version?

### ISSUE THREAD

**Issue by keithbriggs** [cvxgrp/dccp#91]
The docs say:

> You should first install [CVXPY 1.1](http://www.cvxpy.org/).

Does that mean that later versions don't work?

**Comment by SteveDiamond** [comment]
Where is this from? Can you share the link?

**Comment by keithbriggs** [comment]
Sorry, I should have said. It's the README from DCCP. Maybe I posted this in the wrong place.

**Comment by keithbriggs** [comment]
No, I did post it in the right place.  The 1.1 comment is visible at https://github.com/cvxgrp/dccp.
